files-community/Files
# GitHub: Update the deploy Action

## Summary

Current delopy pipeline has only deployment pipeline for Preview branch. But I'm gonna add for Production too.

## Task Checklist

- [ ] Added store config/update/submission steps

## Known Issues

- Not being able to test on fork branch.

## PR Checklist


## Steps To Validate Changes

1. Merge this to the main
2. Run manually
3. Check if succeeded

## Screenshots

_N/A_

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.
